src/js/components/hyperdrive-size/index.js

```diff
@@ -30,8 +30,10 @@ HyperdriveSize.prototype._updateSize = function (state) {
     var s = state.archiveReducer
     if (s.archive && s.archive.content && s.archive.content.bytes) {
       this._size = prettyBytes(s.archive.content.bytes)
+      this.$el.style.display = 'block'
     } else {
       this._size = 0
+      this.$el.style.display = 'none'
```

Comment:
not entirely sure we have to do this right now b/c we still need to design this thing anyway. cc @Kriesse